guide/packages.md

```diff
@@ -106,6 +106,7 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
 | `membrane_sdl_plugin`              | Membrane video player based on SDL   | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_sdl_plugin.svg)](https://hex.pm/packages/membrane_sdl_plugin) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_sdl_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_sdl_plugin)                                                     |
 | `membrane_element_rawvideo_parser` | Plugin for parsing raw video streams | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_element_rawvideo_parser.svg)](https://hex.pm/packages/membrane_element_rawvideo_parser) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_element_rawvideo_parser) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_element_rawvideo_parser) |
 | `membrane_ffmpeg_swscale_plugin`   | Plugin for scaling raw video streams | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_ffmpeg_swscale_plugin.svg)](https://hex.pm/packages/membrane_ffmpeg_swscale_plugin) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_ffmpeg_swscale_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_ffmpeg_swscale_plugin)  |
+ `membrane_video_merger_plugin`   | Plugin for cutting and merging raw video streams | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_video_merger_plugin.svg)](https://hex.pm/packages/membrane_video_merger_plugin)[![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_video_merger_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_video_merger_plugin)  |
```

Comment:
Something is off with formatting here
![image](https://user-images.githubusercontent.com/2119556/128386871-866c8749-55aa-452d-8d65-66b57a24b194.png)
EDIT: And I think I know what:
```suggestion
 `| membrane_video_merger_plugin`   | Plugin for cutting and merging raw video streams | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_video_merger_plugin.svg)](https://hex.pm/packages/membrane_video_merger_plugin)[![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_video_merger_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_video_merger_plugin)  |
```

Reply:
cc: @DominikWolek